### PR TITLE
[PLT-6288] add way to view in the browser and not download in apiV4

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Add a possibility to view the file in the browser and not download directly.
This is just for GET /file apiV4

#### Ticket Link
Jira: https://mattermost.atlassian.net/browse/PLT-6288
GitHub: https://github.com/mattermost/platform/issues/6061

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)

### CODE REVIEW

**Comment by cpanato** [comment]
@jwilander can we spin a server to test if this is the correct behavior?

**Comment by cpanato** [comment]
@jwilander @jasonblais please take a look and do some tests :)

**Comment by cpanato** [comment]
if its ok, i will add some tests to check the headers

**Comment by cpanato** [comment]
@jasonblais @jwilander on my side its ok, if you guys are fine, I will just add some tests

**Comment by jasonblais** [comment]
@jwilander can you help take a look?

File: api4/file.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func InitFile() {
 	l4g.Debug(utils.T("api.file.init.debug"))
 
 	BaseRoutes.Files.Handle("", ApiSessionRequired(uploadFile)).Methods("POST")
-	BaseRoutes.File.Handle("", ApiSessionRequired(getFile)).Methods("GET")
+	BaseRoutes.File.Handle("", ApiSessionRequiredTrustRequester(getFile)).Methods("GET")
 	BaseRoutes.File.Handle("/thumbnail", ApiSessionRequired(getFileThumbnail)).Methods("GET")
 	BaseRoutes.File.Handle("/link", ApiSessionRequired(getFileLink)).Methods("GET")
 	BaseRoutes.File.Handle("/preview", ApiSessionRequired(getFilePreview)).Methods("GET")
```
Comment (jwilander) [inline api4/file.go]: Can you change the /thumbnail and /preview handlers to be `ApiSessionRequiredTrustRequester` as well? And add the query param for those as well

Comment (cpanato) [inline api4/file.go]: done

**Comment by jwilander** [comment]
Tested it out, looks good.

**Comment by grundleborg** [comment]
Thanks @cpanato - looks good.

**Pull request merged**